### ISSUE TITLE
Fanart.tv Clearlogos & ClearArt

### ISSUE THREAD

**Issue by lukehansford** [drinfernoo/plugin.video.openmeta#4]
Can Fanart.tv clearlogos and clearart be added?

Also can you make it so Trakt progress has this enabled.

**Comment by TheWizard67** [comment]
I agree Fanart.tv clearlogos and clearart would be amazing to have! Hope its possible. 👍
